components/diagnosis-suggestion/api/pom.xml

```diff
@@ -84,4 +84,28 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>clirr-maven-plugin</artifactId>
+        <configuration>
+          <ignored> 
+            <!-- Remove the following ignores after we release 1.2m2 -->
+            <difference>
```

Comment:
You should add a comment explaining why this changed, or even better, a `<justification>` tag:

```
<justification>The API should accept both ontology terms and non-standard features.</justification>